### ISSUE TITLE
Ability to set NodePort IP manually

### ISSUE THREAD

**Issue by andersbogsnes** [dask/dask-kubernetes#348]
I love the new setup for KubeCluster with the in-cluster Scheduler. We have the same usecase of wanting to run Dask jobs on remote K8s cluster from laptops and I pretty much have it running in the static version. 

Our blocker for transitioning to KubeCluster is that we need to use Nodeports for accessing the scheduler, but our service accounts aren't allowed to have the "list node" privilege. Is there a way to pass in the IP to use for the scheduler, bypassing the NodePort lookup?

**Comment by jacobtomlinson** [comment]
There isn't a way to do this currently, but we could definitely add this.

We would need a new config option which would make its way here and set the host instead of doing the lookup.

https://github.com/dask/dask-kubernetes/blob/6da9413cdfeac686217ccae3ae29189e04e16144/dask_kubernetes/utils.py#L56-L57

Do you have any interest in contributing this?

**Comment by andersbogsnes** [comment]
Certainly, I'd love to give it a shot! Imagine I should add it to the Kubernetes config and check for it in the lookup function
